v3/vrs_p_r.py

```diff
@@ -64,10 +64,14 @@ def handle_xlsx(self, workbook: Workbook) -> ImportSourceResult:
         worksheet = workbook.active
         mapping = self.get_mapping_by_header(next(worksheet.rows))
 
-        validation_exceptions: list[ImportParkingSiteException] = []
+        static_parking_site_errors: list[ImportParkingSiteException] = []
         static_parking_site_inputs: list[StaticParkingSiteInput] = []
 
         for row in worksheet.iter_rows(min_row=2):
+            # ignore empty lines
+            if row[0].value is None:
+                continue
+
```

Comment:
Could you describe what's happening here? Has the vrs_p_r file empty rows?

Reply:
I had the situation that LibreOffice just added empty rows at the end of the file, which ended up in lot's of errors, because empty lines don't validate.